### ISSUE TITLE
Breast feeding in female

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10076]
**Preferred term label:**
Breast feeding in female

**Synonyms**

**Definition (free text, please give PubMed ID)**
This describes the sequalae of breastfeeding in the female, as opposed to https://hpo.jax.org/app/browse/term/HP:4000122, which describes sequalae in the child. The language of both should make the distinction more clear.
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Galactocele

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
just to be clear - Do we mean "History of having breastfed" ?

**Comment by MickeySegal** [comment]
History in a woman of breastfeeding a child recently, predisposing to Galactocele and Breast abscess

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001045
- label: History of recent breastfeeding
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: History of recently breastfeeding a child.
- comment: Recent breastfeeding can be a risk factor for galactocele and breast abscess.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10076
- synonyms:
